package.json

```
@@ -13,7 +13,7 @@
     "lint": "standard",
     "lint:fix": "standard --fix",
     "test:typescript": "tsd",
-    "test:unit": "tap",
+    "test:unit": "node --test",
```

We should add the coverage too:

```suggestion
    "test:unit": "c8 --100 node --test",
```

---

also means adding c8 as devDependency

---

@Eomm the `--100` flag is wishful thinking 😄 just added `c8`